main.go

```diff
@@ -1,4 +1,4 @@
-package types
+package openRuntimes
```

Comment:
```suggestion
package openruntimes
```
go conventions do not recommend using camelCase for package names.

Reply:
Nice catch! 🔥